doc/guides/west/manifest.rst

```diff
@@ -840,44 +836,50 @@ using ``glob`` syntax.
      projects:
        - name: app
          url: https://git.example.com/mainline/app
-       - name: library
-         url: https://git.example.com/mainline/library
-         revision: refs/heads/only-in-mainline
-       - name: library2
-         url: https://git.example.com/mainline/library-2
-       - name: unnecessary-project
-         url: https://git.example.com/mainline/unnecessary-project
```

Comment:
Why is this removed ?
Without this, I don't think this example is useful, as nothing is exluded during import.

Reply:
The `lib` project is excluded, no?

Reply:
This is the example manifest:
```
# mainline west.yml:
   manifest:
     projects:
       - name: app
         url: https://git.example.com/mainline/app
      - name: lib
        path: libraries/lib
        url: https://git.example.com/mainline/lib
       - name: lib2
         path: libraries/lib2
         url: https://git.example.com/mainline/lib2

  # downstream west.yml:
   manifest:
     projects:
       - name: mainline
         url: https://git.example.com/mainline/manifest
         import:
           path-whitelist: libraries/*
```

and as both `lib` and `lib2` are located under the folder `libraries` and that the path `libraries` is given in `path-whitelist: libraries/*`, then both of them are included, and nothing is excluded. 

Therefore, to show a project being excluded in this example, you'll need the `unnecessary-project`

Reply:
Just noticed. Project `app` is actually excluded.
And in that case `unnecessary-project` is not needed.

(But `lib` is still included)